README.md

```diff
@@ -8,6 +8,10 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/KohlsTechnology/eunomia)](https://goreportcard.com/report/github.com/KohlsTechnology/eunomia)
 [![codecov](https://codecov.io/gh/KohlsTechnology/eunomia/branch/master/graph/badge.svg)](https://codecov.io/gh/KohlsTechnology/eunomia)
 
+## Getting started
+
+If you want to deploy Eunomia from scratch to a local Minikube cluster, begin with the [Getting started](./GETTING_STARTED.md) document. It will guide your through the setup step-by-step.
+
```

Comment:
The hierarchy section mentions "You can also use the relative path `./`, which means it'll also load the variables defined in `contextDir` directly (same folder that as `hierarchy.lst`)." How does this fit into priority? Would it be first or last priority when compared against the hierarchy.lst?

Reply:
It depends where you put it in the sequence - in [this example](https://github.com/KohlsTechnology/eunomia/blob/master/examples/hello-world-hierarchy/demo/hierarchy.lst) `./` has the highest priority.
But it's a good idea to clarify it